Utilities/bootstrap

```diff
@@ -229,34 +231,9 @@ def get_ninja_path(args):
 
 def get_build_target(args):
     """Returns the target-triple of the current machine."""
-    if platform.system() == 'Darwin':
-        return "x86_64-apple-macosx"
-    elif platform.system() == 'Linux':
-        if 'ANDROID_DATA' in os.environ:
-            if platform.machine().startswith('armv7'):
-                return 'armv7a-unknown-linux-androideabi'
-            elif platform.machine() == 'aarch64':
-                return 'aarch64-unknown-linux-android'
-            else:
-                raise SystemExit("ERROR: unsupported Android platform:", platform.machine())
-        elif platform.machine() == 'x86_64':
-            return "x86_64-unknown-linux-gnu"
-        elif platform.machine() == "i686":
-            return "i686-unknown-linux"
-        elif platform.machine() == 's390x':
-            return "s390x-unknown-linux"
-        elif platform.machine() == 'ppc64le':
-            return 'powerpc64le-unknown-linux'
-        elif platform.machine().startswith("armv7"):
-            return 'armv7-unknown-linux-gnueabihf'
-        elif platform.machine() == 'aarch64':
-            return 'aarch64-unknown-linux'
-        else:
-            raise SystemExit("ERROR: unsupported machine:", platform.machine())
-    elif platform.system() == 'Windows':
-        return 'x86_64-unknown-windows-msvc'
-    else:
-        raise SystemExit("ERROR: unsupported system:", platform.system(), platform.machine())
+    target_info_json = call_output([args.swiftc_path, '-print-target-info'], verbose=args.verbose)
```

Comment:
Can you add a fallback here on Darwin because I want SwiftPM to be compatible with latest version of Xcode for easy contributions.

Reply:
Unsure what you mean, never used XCode.

Reply:
Basically, if `swift -print-target-info` fails to execute and the platform is Darwin, fall back to hardcoded value of "x86_64-apple-macosx".